inference-engine/src/gna_plugin/backend/dnn_components.hpp

```diff
@@ -12,11 +12,21 @@
 
 namespace GNAPluginNS {
 namespace backend {
+
+struct DnnComponentExtra {
```

Comment:
What is the purpose of introducing this class? Wound't it be better to use std::tuple instead of std::pair in DnnComponents?
Alternatively I would suggest better name for this class.